plugins/MobileAppType/plugin.json

```diff
@@ -0,0 +1,4 @@
+{
+  "name": "MobileAppType",
+  "description": "Allows you to create Mobile Apps"
```

Comment:
suggested: `Analytics for Mobile: lets you create Mobile Apps properties in Piwik, and measure user interactions with a SDK for iOS, Android or Titanium.`